cluster/manifests/skipper/hpa.yaml

```diff
@@ -53,6 +53,25 @@ spec:
       target:
         averageValue: {{ .ConfigItems.skipper_scaling_schedule_hyped_articles_target }}
         type: AverageValue
+{{ end }}
+{{ with $config_items := .ConfigItems }}
+	{{ range $i := split "1 2 3" " " }}
+		{{ $name := index $config_items (printf "skipper_scaling_schedule_name_%s" $i) }}
+		{{ $target := index $config_items (printf "skipper_scaling_schedule_target_%s" $i) }}
+		{{ if and (ne $name "") (ne $target "") }}
```

Comment:
I wonder if a little copy is not simpler than this. :)
We also do not win anything, because if we need "4" than we need to add a "4" here....

Reply:
I think I'll do like @mikkeloscar suggests because we use such encoding elsewhere, e.g.
https://github.com/zalando-incubator/kubernetes-on-aws/blob/dc0d37ed3e2c86c7ca462d5fe2e6e5e2cd3a35f4/cluster/node-pools/worker-combined/stack.yaml#L88-L95